metashare/api/models.py

```diff
@@ -504,11 +552,10 @@ def finalize_status_completed(self, *, originating_user_id):
         self.status = TASK_STATUSES.Completed
         self.has_unmerged_commits = False
         self.pr_is_open = False
+        self.project.has_unmerged_commits = True
+        # This will save the project, too:
```

Comment:
This might not actually trigger a project.save()? Could pass through a `force_project_save` kwarg or something